### PR TITLE
Node 20 + Yarn 4

### PR DESCRIPTION
**Pull request by ahuth**
This PR updates Node to v20 and Yarn to v4.

### CODE REVIEW

File: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs [hunk 1]
```diff
@@ -1,28 +0,0 @@
-/* eslint-disable */
-//prettier-ignore
-module.exports = {
-name: "@yarnpkg/plugin-workspace-tools",
```
Comment (ahuth) [inline .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs:4]: The workspace tools are now built in to Yarn itself.

File: .yarnrc.yml [hunk 2]
```diff
@@ -1,9 +1,7 @@
-nodeLinker: node-modules
+compressionLevel: mixed
```
Comment (ahuth) [inline .yarnrc.yml:1]: The `yarn set version latest` command made these changes.

Interestingly, the default for `compressionLevel` is 0, not mixed, per https://yarnpkg.com/configuration/yarnrc#compressionLevel 🤔

**Review by booc0mtaco — APPROVED** [review]
I forget the command to use to install the latest version of `node`. for `nodenv` users, what should we run?

**Comment by ahuth** [comment]
Thanks 🙏

**Pull request merged**